versioned/tiered/dynamodb/src/test/java/org/projectnessie/versioned/impl/condition/TestExpressions.java

```diff
@@ -64,106 +54,94 @@ void multiAlias() {
 
   @Test
   void aliasReservedWord() {
-    AliasCollectorImpl c = new AliasCollectorImpl();
-    ExpressionPath aliased = ExpressionPath.builder("abort").build().alias(c);
+    final AliasCollectorImpl c = new AliasCollectorImpl();
+    final ExpressionPath aliased = ExpressionPath.builder("abort").build().alias(c);
     assertEquals("#abortXX", aliased.asString());
     assertEquals("abort", c.getAttributeNames().get("#abortXX"));
     assertTrue(c.getAttributesValues().isEmpty());
   }
 
   @Test
   void aliasSpecialCharacter() {
-    AliasCollectorImpl c = new AliasCollectorImpl();
-    ExpressionPath aliased = ExpressionPath.builder("foo.bar").build().alias(c);
+    final AliasCollectorImpl c = new AliasCollectorImpl();
+    final ExpressionPath aliased = ExpressionPath.builder("foo.bar").build().alias(c);
     assertEquals("#f0", aliased.asString());
     assertEquals("foo.bar", c.getAttributeNames().get("#f0"));
     assertTrue(c.getAttributesValues().isEmpty());
   }
 
   @Test
   void aliasValue() {
-    AliasCollectorImpl c = new AliasCollectorImpl();
-    Value v0 = Value.of(av0);
-    Value v0p = v0.alias(c);
+    final AliasCollectorImpl c = new AliasCollectorImpl();
+    final Value v0 = Value.of(av0);
+    final Value v0p = v0.alias(c);
     assertEquals(":v0", v0p.getPath().asString());
     assertEquals(AttributeValueUtil.fromEntity(av0), c.getAttributesValues().get(":v0"));
     assertTrue(c.getAttributeNames().isEmpty());
   }
 
   @Test
   void equals() {
-    ExpressionFunction f = ExpressionFunction.equals(ExpressionPath.builder("foo").build(), av0);
-    AliasCollectorImpl c = new AliasCollectorImpl();
-    ExpressionFunction f2 = f.alias(c);
+    final ExpressionFunction f = ExpressionFunction.equals(ExpressionPath.builder("foo").build(), av0);
+    final AliasCollectorImpl c = new AliasCollectorImpl();
+    final ExpressionFunction f2 = f.alias(c);
     assertEquals("foo = :v0", f2.asString());
     assertEquals(AttributeValueUtil.fromEntity(av0), c.getAttributesValues().get(":v0"));
   }
 
   @Test
   void listAppend() {
-    Value v0 = ExpressionFunction.appendToList(ExpressionPath.builder("p0").build(), Entity.ofList(av0));
-    AliasCollectorImpl c = new AliasCollectorImpl();
-    Value v0p = v0.alias(c);
+    final Value v0 = ExpressionFunction.appendToList(ExpressionPath.builder("p0").build(), Entity.ofList(av0));
+    final AliasCollectorImpl c = new AliasCollectorImpl();
+    final Value v0p = v0.alias(c);
     assertEquals("list_append(p0, :v0)", v0p.asString());
     assertEquals(AttributeValueUtil.fromEntity(av0), c.getAttributesValues().get(":v0").l().get(0));
   }
 
-  @Test
```

Comment:
Finally figured out why this is removed. It's odd that this is removed here when AddClause isn't removed in this patch but by a different one, I think you should revert changes to this file.